README.md

```diff
@@ -38,3 +38,22 @@ Don't do link check:
 ```Bash
 $ npx respec-validate --no-links spec.html
 ```
+
+## FAQ
+
+### Can I make it run faster?
+
+If you already have Chromium installed, you can set the following environment variables to avoid downloading Chromium unnecessarily.
+
+``` bash
+export PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=1
+# Replace the path with your actual Chromium browser path
+export PUPPETEER_EXECUTABLE_PATH=/usr/bin/google-chrome
```

Comment:
This will be `/Applications/Google Chrome.app/Contents/MacOS/Google Chrome` on macOS if I'm right. Add this also?

Reply:
Should be:
"./Applications/Google Chrome.app/Contents/MacOS/Google chrome"

Might as well add it...

Reply:
Chrome with lowercase c? Stack overflow told otherwise..

Reply:
odd... on my computer is lower... it doesn't matters, as basically nobody uses case sensitive FS.